docs/cover-all-tiles.md

```diff
@@ -0,0 +1,80 @@
+# cover All Tiles
+
+Cover all of the tiles of one image with a different tile.
+
+```sig
+tileUtil.coverAllTiles(null, null)
+```
+
+## Parameters
+
+* **tileKind**: the image of the tiles in the tilemap to cover with another tile.
+* **cover**: the image of the tile to cover with.
+
+## Example
+
+Set a checkerboard tilemap with brown and white tiles into the game scene. When button `A` is pressed, cover the white tiles with the blac tiles and cover the brown tiles with transparent tiles.
+
+```blocks
+controller.A.onEvent(ControllerButtonEvent.Pressed, function () {
+    tileUtil.coverAllTiles(assets.tile`myTile0`, assets.tile`myTile2`)
+    tileUtil.coverAllTiles(assets.tile`myTile1`, assets.tile`transparency16`)
+})
+tiles.setCurrentTilemap(tilemap`level1`)
+```
+
+## See also
+
+[replace all tiles](/tile-util/replace-all-tiles),
```

Comment:
Not sure how the see also paths will fit into the doc space. Might need to remove them if it doesn't work out.